organization/search_indexes.py

```diff
@@ -0,0 +1,205 @@
+from haystack import indexes
+
+from search.base_search_indexes import SearchEntity
+from sfm_pc.templatetags.countries import country_name
+
+
+class OrganizationIndex(SearchEntity, indexes.Indexable):
+
+    '''
+    {
+        'id': org_id,
+        'entity_type': 'Organization',
+        'content': content,
+        'location': '',  # disabled until we implement map search
+        'published_b': organization.published,
+        'country_ss': countries,
+        'division_id_ss': division_ids,
+        'start_date_dt': first_cited,
+        'end_date_dt': last_cited,
+        'open_ended_s': open_ended,
+        'organization_name_s': name,
+        'organization_parent_name_ss': parent_names,
+        'organization_parent_count_i': parent_count,
+        'organization_membership_ss': memberships,
+        'organization_classification_ss': classes,
+        'organization_classification_count_i': class_count,
+        'organization_alias_ss': aliases,
+        'organization_headquarters_s': hq,
+        'organization_exact_location_ss': exactloc_names,
+        'organization_site_count_i': last_site_exists,
+        'organization_country_count_i': country_count,
+        'organization_area_ss': areas,
+        'organization_start_date_dt': first_cited,
+        'organization_end_date_dt': first_cited,
+        'organization_adminlevel1_ss': list(admin_l1_names),
+        'text': content
+    }
+    '''
+
+    CONTENT_FIELDS = (
+        'name',
+        'aliases',
+        'classifications',
+        'headquarters',
+        'parent_names',
+        'countries',
+        'exact_locations',
+        'areas',
+    )
+
+    open_ended = indexes.CharField()
+    name = indexes.CharField()
+    parent_names = indexes.MultiValueField(faceted=True)
+    memberships = indexes.MultiValueField(faceted=True)
+    classifications = indexes.MultiValueField(faceted=True)
+    aliases = indexes.MultiValueField()
+    headquarters = indexes.CharField()
+    exact_locations = indexes.MultiValueField()
+    areas = indexes.MultiValueField()
+    adminlevel1s = indexes.MultiValueField(faceted=True)
+
+    def get_model(self):
+        # For some reason, Haystack doesn't want to discover the index if we
+        # import the model into the global namespace for this module. So,
+        # import it here.
+        from organization.models import Organization
+
+        return Organization
+
+    def prepare(self, object):
+        self.prepared_data = super().prepare(object)
+
+        self.prepared_data['content'] = self._prepare_content(self.prepared_data)
+        self.prepared_data['countries'] = self._prepare_countries(self.prepared_data)
+
+        return self.prepared_data
+
+    def _prepare_countries(self, prepared_data):
+        countries = set()
+
+        for division in prepared_data['division_ids']:
+            countries.update([country_name(division)])
```

Comment:
i understand this is copy paste, but this seems like it should be

`countries.add(country_name(division))`

or even better

`
countries = {country_name(division) for division in prepared_data['division_ids']}
`